source/features/quick-unplanned.tsx

```diff
@@ -0,0 +1,75 @@
+import delegate, {DelegateEvent} from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import select from 'select-dom';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+const id = features.getFeatureID(import.meta.url);
+
+const unplannedCheckbox = 'input[name="state_reason"][value="not_planned"]';
+
+// TODO: Replace with https://github.com/fregante/select-dom/issues/20
+// @ts-expect-error idc, temporary
+const selectExpect: typeof select = (selector: string, context?: Element) => {
+	const element = select(selector, context);
+	if (element) {
+		return element;
+	}
+
+	features.log.error(import.meta.url, `Expected element not found:\n${selector}`);
+};
+
+function update(dropdown: HTMLElement): void {
+	const form = dropdown.closest('form')!;
+	const radio = selectExpect(unplannedCheckbox, dropdown);
+	const mainButton = selectExpect('[name="comment_and_close"]', form);
+	const icon = selectExpect('.octicon-skip', dropdown);
```

Comment:
This feature is front and center, we need to make sure it does not fail. This isn't enough either